### ISSUE TITLE
doesn't work with gopro hero 3+

### ISSUE THREAD

**Issue by Frequel** [KonradIT/GoProStream#21]
Traceback (most recent call last):
  File "/usr/lib/python3.6/http/client.py", line 283, in _read_status
    status = int(status)
ValueError: invalid literal for int() with base 10: 'ÿûAllow:'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "GoProStream.py", line 118, in <module>
    gopro_live()
  File "GoProStream.py", line 47, in gopro_live
    response_raw = urllib.request.urlopen('http://10.5.5.9/gp/gpControl').read().decode('utf-8')
  File "/usr/lib/python3.6/urllib/request.py", line 223, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.6/urllib/request.py", line 526, in open
    response = self._open(req, data)
  File "/usr/lib/python3.6/urllib/request.py", line 544, in _open
    '_open', req)
  File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.6/urllib/request.py", line 1346, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/usr/lib/python3.6/urllib/request.py", line 1321, in do_open
    r = h.getresponse()
  File "/usr/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/usr/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.6/http/client.py", line 287, in _read_status
    raise BadStatusLine(line)
http.client.BadStatusLine: HTTP/1.1 ÿûAllow: GET 

i had this error when i run the python script, instead with a gopro hero4 it works

**Comment by KonradIT** [comment]
Run this code instead:

```
PASSWORD=urllib.request.urlopen("http://10.5.5.9/bacpac/sd").read()
Password =  str(PASSWORD, 'utf-8')
text=re.sub(r'\W+', '', Password)
urllib.request.urlopen("http://10.5.5.9/camera/PV?t=" + text + "&p=%02")
subprocess.Popen("ffplay " + URL, shell=True)
```

**Comment by Frequel** [comment]
now it works but in the terminal i have the list of connection/frame that the cam is streaming, not a real problem but this thing doesn't happen with the hero 4

**Comment by KonradIT** [comment]
maybe a UDP/hls difference

**Comment by Frequel** [comment]
I managed to fix this in #22
